Core/include/Acts/Vertexing/GaussianTrackDensity.hpp

```diff
@@ -34,16 +34,20 @@ class GaussianTrackDensity {
     /// @param c2_ Quadratic coefficient in exponent
     /// @param lowerBound_ The lower bound
     /// @param upperBound_ The upper bound
+    /// @param time_ Time component 
     TrackEntry(double z_, double c0_, double c1_, double c2_,
-               double lowerBound_, double upperBound_)
+               double lowerBound_, double upperBound_,double time_)
```

Comment:
To me it looks a bit like we would need more coefficients by adding time since there are `z` coefficients